lib/Value/Real.pm

```diff
@@ -195,7 +195,7 @@ sub atan2 {
 ##################################################
 
 sub string {
-  my $self = shift; my $equation = shift; my $prec = shift;
+  my $self = shift; my $equation = shift; shift; shift; my $prec = shift;
```

Comment:
The extra shifts seem to be for "open" and "close" which some such routines need.